CHANGELOG.rst

```diff
@@ -3,6 +3,12 @@ Change Log
 
 * :+1: Editing the `title` and `meta` of a `Widget` can now be performed simultaneously.
 
+Backwards incompatible changes
+------------------------------
+
+* Deleted `pykechain/models/customization.py`
```

Comment:
```suggestion
* As planned and marked with a PendingDeprecationWarning we deprecate the customizations of Activities. This can only be done with old KE-chain versions which are no longer available in production. 
```